website/cue/reference/configuration.cue

```diff
@@ -7,6 +7,21 @@ configuration: {
 
 configuration: {
 	configuration: {
+		acknowledgements: {
+			common:      true
+			description: "Controls how acknowledgements are handled by all sources. These settings may be overridden in individual sources."
+			required:    false
+			type: object: options: {
+				enabled: {
+					common:      true
+					description: "Controls if sources will wait for destination sinks to deliver the events before acknowledging receipt. If set to `true`, all capable sources will have acknowledgements enabled."
```

Comment:
```suggestion
					description: "Controls if sources will wait for destination sinks to deliver the events, or persist them to a disk buffer, before acknowledging receipt. If set to `true`, all capable sources will have acknowledgements enabled."
```

Reply:
Might also be worth linking to the acknowledgement docs here once they exist too.

Reply:
Agreed.